src/components/Table/components/TableBody/test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+import Table from '../../index';
+import TableBody from './index';
+import TableRow from '../TableRow';
+import Typography from '../../../Typography';
+
+
+describe('Table Body', () => {
+
+
+    it('Renders correctly', () => {
+        const content = renderer
+            .create((
+                <TableBody>
+                    <TableRow>
+                        <td>Row1-Col1</td><td>Row1-Col2</td><td>Row1-Col3</td>
+                    </TableRow>
+                    <TableRow>
+                        <td>Row2-Col1</td><td>Row2-Col2</td><td>Row2-Col3</td>
+                    </TableRow>
+                </TableBody>
+            ))
+            .toJSON();
+
+        expect(content).toMatchSnapshot();
+    });
+
+
+    it('Includes custom className', () => {
+        const className = `class-${String(Math.floor(Math.random() * 1000))}`;
+        const content = shallow((
```

Comment:
This test case should probably just include the `TableBody` component, because a failure in the rendering of the `Table` or `TableRow` component could probably make this fail unnecessarily.  It would also make this test much simpler in execution

Reply:
Done.